benchmark/storage/garbage_collector_benchmark.cpp

```diff
@@ -98,9 +126,37 @@ BENCHMARK_DEFINE_F(GarbageCollectorBenchmark, ReclaimTime)(benchmark::State &sta
   state.SetItemsProcessed(state.iterations() * num_txns);
 }
 
+/**
+ * Run a large number of updates on a small table to generate contention with the GC. Measure the number of transactions
+ * that the GC managed to free during the workload by subtracting the number of "lagging" transactions that still
+ * remained to be cleaned up by the GC after the workload was done running.
+ */
+// NOLINTNEXTLINE
+BENCHMARK_DEFINE_F(GarbageCollectorBenchmark, HighContention)(benchmark::State &state) {
+  uint64_t lag_count = 0;
+  // NOLINTNEXTLINE
+  for (auto _ : state) {
+    LargeTransactionBenchmarkObject tested({8, 8, 8}, 100, txn_length, update_select_ratio, &block_store_,
```

Comment:
Not in your PR, but we apparently forgot to switch these to using the builders as well. (Tanuj changed the test versions a couple months back)